packages/storybook/src/Tab.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { RenderOptions } from '@storybook/addons';
+import { useStorybookState, useStorybookApi } from '@storybook/api';
+import { config } from './config';
+import { AddonPanel } from '@storybook/components';
+
+const style = {
+  display: 'block',
+  height: '90vh',
```

Comment:
90vh doesn't ever seem to fit the window right, can this be `calc(100vh - _px)` instead?